README.md

```diff
@@ -14,15 +31,19 @@ While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great
 Swift-Kuery is easy to learn, consumable framework that comes with a set of [implemented plugins](#list-of-plugins).
 
 ## Table of Contents
-* [Example](#example)
+* [Examples](#examples)
 * [SQL Injection Prevention using Parameterization](#sql-injection-prevention-using-parameterization)
 * [Prepared Statements](#prepared-statements)
 * [Schema Management](#schema-management)
 * [Query Examples](#query-examples)
```

Comment:
The Query Examples section needs some more explanation than "Let's see more examples of how to build and execute SQL queries using Swift-Kuery." What is it's reason for being, what is the format, not all of the entries have execute statements, why do some and not others? Is it to show how to build more complex SQL queries using Swift-Kuery? Is the format to provide an SQL statement and then to demonstrate how to build the same query in Swift-Kuery?